monai/transforms/utility/dictionary.py

```diff
@@ -64,7 +64,11 @@
     "CastToTyped",
     "ToTensord",
     "ToNumpyd",
+    "ToNumpyD",
```

Comment:
The previous order of this list here is: (1) All the transform names with "d", (2) transform name with "D" and with "Dict".
I think maybe you can keep the original order or change all the transforms to: transform name with "d", with "D" and with "Dict".

Thanks.

Reply:
sure I sorted them alphabetically.